Pearson-Advance/frontend-app-instructors-portal
# feat: Initial config for instructor portal

## Ticket

https://agile-jira.pearson.com/browse/PADV-1144

## Description

In this PR is added the initial config for frontend-instructructors.

![1](https://github.com/Pearson-Advance/frontend-app-instructors-portal/assets/52179095/8894c989-fbc1-4a1c-a4ee-9c23649c0e92)


## Changes made

- Forked repository from openedx [MFE template](https://github.com/openedx/frontend-template-application/tree/open-release/olive.4).
- Remove unnecessary definitions and files.
- Github actions enabled.
- Unnecessary Github Actions have been removed.
- Some dependencies versions were updated in order to make the project work.

## How to test

- Clone the instructor Portal MFE
- Run openedx services
- Run npm run start
- Go to http://http://localhost:1990 and you should be able to see the mfe running
